lib/org/chromium/webidl/IDLFragmentExtractor.js

```diff
@@ -60,9 +61,15 @@ foam.CLASS({
           // Ignoring all tags. Only extracting text content within pre tags.
           if (isTag) {
             if (item.nodeName === 'pre') {
+              // Finished matching pre tag at this point.
               tagMatching = true;
               tagStack.pop();
-              idlFragments.push(lexer.lib.unescapeString(content));
+
+              if (ignoreNextPre) {
+                ignoreNextPre = false;
```

Comment:
Do we need to process other IDL fragments after the index, or does it appear at the end of the document?